app/src/components/v-table/table-header.vue

```diff
@@ -278,7 +278,7 @@ function toggleManualSort() {
 		padding: 0 12px;
 		font-weight: 500;
 		font-size: 14px;
-		background-color: var(--v-table-background-color);
+		background-color: var(--v-table-background-color, transparent);
```

Comment:
Seems like this shouldn't be transparent, as the table header needs to have a background to _overlay_ on top of the rows when it's stickied. Looks like it was unintentionally turned transparent via https://github.com/directus/directus/pull/20311/files#diff-7d46deff96a55a4e167da2333fdf258014408c30af67b5cc5f88fd90b845999eL351-R351 as it used to use input background as the background color.

Using the old value `var(--theme--form--field--input--background)` does seem to fix it, but since input background actually derive it's value from theme background page, I wonder should the following suggestion be the correct approach instead:

```suggestion
		background-color: var(--v-table-background-color, var(--theme--background-page));
```